### PR TITLE
Templates

### LINKED ISSUE #215: Website and process for meeting notes + org repo
<sup>_This initial comment is collaborative and open to modification by all._</sup>
## Task Summary

🎟️ **Re-ticketed from:** #  
🗣 **Loomio:** N/A  
📅 **Due date:** N/A  
🎯 **Success criteria:** ...  

I would like to propose a replacement of our current [meetings index](link.hypha.coop/meetings), which I think has the following shortcomings:

- The pipeline of `hackmd -> PR -> github` issue requires a lot of manual overhead
- We don't make good use of templates, lots of manual editing, human errors have caused confusions in the past
- Does not handle private pads well (private hackmds are still public)
- We leave a lot of old hackmds (with pre-PR'd content) hanging even after we migrate to github, and often times index is never updated (bc it's manual and no one remembers to update it after the PR gets merged)
- Index is confusing and inconsistent
- Hard to tell what is in source control, what is in hackmd, and what is private vs. public
- Pages are slow to load
- Poor access control, anyone outside the org can edit our content
- Not easy to present to public / partner orgs

The proposed replacement is already mostly done, and should address all the issues above.

org

It uses the default jekyll theme with a custom _post_ layout, so when a new notes gets merged a new github page is built on the organizing repo. No modification of the notes need to be made (unless we want to add a tag to categorize them by WG).

With this new approach, a meeting will go like this:

- Each wg / meeting type gets a custom access-restricted hackmd which is recycled per meeting
- `@roobot <wg> reset` will reset the recyclable pad based on template
- `@roobot <wg> public` will file a PR to _organizing_
- `@roobot <wg> private` will file a PR to _organizing-private_ and create a stub PR to _organizing_ that says "This note is private, linking the post to the private repo's .md file", this way we have a global index that shows also private notes but through a access controlled link

In this model, we can decide access level per meeting (it's possible that a specific bizdev can be public and a governance needs to be private, although they usually aren't). The manual overhead is much less, and with little room for error bc we offload that copy-paste work to template + roobot. Everything will be in source control, there are no out-of-sync loose pads hanging, we have a fast website to share externally and no one can change the content. We can use frontmatter + liquid to build better categorize notes in the auto-built index and even add entries for retreat, interviews, resolutions, etc.

I am happy to work with @hyphacoop/infra-wg to get this implemented. As I mentioned, most of the work is already done and on my local.

## To Do

- [x] Clean up jekyll site and send PR to organizing
- [x] Update templates and create privileged hackmd pad per working group / meeting type
- [ ] ~~Update roobot to do these new tasks~~ (edit: https://github.com/hyphacoop/organizing/issues/222)
- [x] Set up https://meetings.hypha.coop and remap shortlink to it
- [x] Update handbook references

### PR DESCRIPTION
**Pull request by benhylau**
Resolves #215 addressing all remaining feedback.

Screen Shot 2020-03-13 at 12 06 33 AM

### CODE REVIEW

**Comment by benhylau** [comment]
I also think this makes the roobot task #222 very low priority, because the process seems really easy. I can explain at next meeting.

File: -meeting-template.md [hunk 1]
```diff
@@ -1,62 +1,66 @@
...
-robots: noindex, nofollow, noarchive, nocache
 ---
-# YYYY-MM-DD Hypha Worker Co-op: All Hands Meeting
+# 2020-MM-DD Hypha Worker Co-op: All Hands
 
-<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
+<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][drive]</sup>
 
-Time:       ...  
+Time:       5-5:30pm ET
```
Comment (dcwalk) [inline -meeting-template.md]: This should be a full hour

File: -meeting-template.md [hunk 2]
```diff
@@ -88,11 +92,18 @@ Notetaker:  ... :raising_hand: Next up: COPY FROM PAST WEEK
...
 [biz-wg]: https://link.hypha.coop/biz-wg
 [fin-wg]: https://link.hypha.coop/fin-wg
 [gov-wg]: https://link.hypha.coop/gov-wg
+[inf-wg]: https://link.hypha.coop/inf-wg
 [ops-wg]: https://link.hypha.coop/ops-wg
+
+<!-- Links: Archive -->
+[public]:   https://github.com/hyphacoop/organizing/new/master?filename=_posts/meeting-notes/2020-MM-DD-all-hands.md
+[index]:    https://github.com/hyphacoop/organizing/new/master?filename=_posts/private/meeting-notes/2020-MM-DD-all-hands.md&value=https://github.com/hyphacoop/organizing/new/master?filename=_posts/meeting-notes/2020-MM-DD-all-hands.md&value=Oops%21%20The%20notes%20from%20%2A%2A%7B%7B%20page.date%20%7C%20date%3A%20%22%25Y-%25m-%25d%22%20%7D%7D%20%7B%7B%20page.title%20%7D%7D%2A%2A%20is%20not%20open%20to%20public%20%3Alock%3A%0A%0AIf%20you%20have%20privileged%20access%2C%20you%20can%20%3Ca%20href%3D%22%7B%7B%20site.organizing-private%20%7D%7D%2F%7B%7B%20page.path%20%7C%20remove%3A%20%27_posts%2Fprivate%2F%27%20%7D%7D%22%3E%2A%2Aview%20the%20notes%20here%2A%2A%3C%2Fa%3E%20%3Akey%3A%0A%0APlease%20see%20the%20_Working%20Open_%20guidelines%20in%20our%20%5BHandbook%5D%28https%3A%2F%2Fhandbook.hypha.coop%2Fworking-open.html%29.
```
Comment (dcwalk) [inline -meeting-template.md]: This link...?

File: index.md [hunk 3]
```diff
@@ -9,4 +9,7 @@ title: About
 [Hypha](https://hypha.coop) is a worker co-operative based out of Ontario, Canada.
 We used this website to index our coordination notes from [hyphacoop/organizing](https://github.com/hyphacoop/organizing).
 
-This website is created with [jekyll](https://github.com/jekyll/jekyll) with the [minima](https://github.com/jekyll/minima) theme.
\ No newline at end of file
+During our virtual meetings, we use [HackMD](https://hackmd.io) to take notes collaboratively with :seedling: in-progress pads that are [accessible to members only](https://hackmd.io/VN4ltoJJTwWSVdODqfBf2w).
+These meeting notes are then reviewed and archive on [GitHub](https://https://github.com/hyphacoop/organizing) with open or :lock: restricted access according to the _Working Open_ guidelines in our [Handbook](https://handbook.hypha.coop/working-open.html).
```
Comment (dcwalk) [inline index.md]:
```suggestion
These meeting notes are then reviewed and archive on [GitHub](https://https://github.com/hyphacoop/organizing) with open or restricted  :lock: access according to the _Working Open_ guidelines in our [Handbook](https://handbook.hypha.coop/working-open.html).
```

File: -meeting-template.md (hunk 1 above)
Comment (benhylau) [reply -meeting-template.md]: Good catch!

File: -meeting-template.md (hunk 2 above)
Comment (benhylau) [reply -meeting-template.md]: Yup it's good! Try it :D

**Pull request merged**